README.md

```diff
@@ -227,15 +227,24 @@ python examples/run_smarts.py --algo SAC --scenario ./scenarios/loop --n_agents
 If you're comfortable using docker or are on a platform without suitable support to easily run SMARTS (e.g. an older version of Ubuntu) you can run the following,
 
 ```bash
-docker run --rm -it -v $(PWD):/src -p 8081:8081 huaweinoah/smarts:<version>
-# E.g. docker run --rm -it -v $(PWD):/src -p 8081:8081 huaweinoah/smarts:v0.4.3-pre
+$ cd /path/to/SMARTS
+$ docker run --rm -it -v $PWD:/src -p 8081:8081 huaweinoah/smarts:<version>
+# E.g. docker run --rm -it -v $PWD:/src -p 8081:8081 huaweinoah/smarts:v0.4.12
 # <press enter>
 
 # Run Envision server in the background
+# This will only need to be run if you want visualisation
 $ scl envision start -s ./scenarios -p 8081 &
+
+# Build an example
+# This needs to be done the first time and after changes to the example
+$ scl scenario build scenarios/loop --clean
```

Comment:
Are the users supposed to enter the docker container to execute these commands? if so we can provide the lines for entering docker container as well

Reply:
`$ docker run --rm -it -v $PWD:/src -p 8081:8081 huaweinoah/smarts:<version>`

Starts the interactive console, we assume that they have ran that.